packages/flutter/lib/src/material/material_state.dart

```diff
@@ -684,5 +684,11 @@ class MaterialStatePropertyAll<T> implements MaterialStateProperty<T> {
   T resolve(Set<MaterialState> states) => value;
 
   @override
-  String toString() => 'MaterialStatePropertyAll($value)';
+  String toString() {
+    if (value is double || value is double?) {
```

Comment:
isn't `value is double` always true when `value is double?` is true? Wouldn't it be enough to just check for `value is double?`?